roles/ucp/tasks/main.yml

```diff
@@ -32,6 +32,10 @@
     - "{{ ucp_instance_id_file }}"
   when: node_name != ucp_bootstrap_node_name
 
+- name: copy the ucp license to the remote machine
+  copy: src={{ ucp_license_file }} dest={{ ucp_license_remote }}
```

Comment:
Yes, the first node can be logged-into without applying the license, but no nodes can join the cluster etc.